src/python/mod.rs

```diff
@@ -164,7 +164,7 @@ impl Dataset {
         let arr = arr.downcast::<PyArrayDyn<T>>().unwrap();
 
         let mut v = unsafe { arr.as_array_mut() };
-        v.par_mapv_inplace(|v| if v == cond { fv } else { v });
+        v.mapv_inplace(|v| if v == cond { fv } else { v });
```

Comment:
This makes the check run sequentially (it makes a big difference for encoded variables in e.g. xarray). It should be provided by `use ndarray::parallel::prelude::*;` but I think that requires netcdf, hdf, and ndarray to be the same version.

Reply:
I opened https://github.com/PyO3/rust-numpy/pull/439 to get the newest `ndarray` in `numpy`. As a hack we could depend on `ndarray=0.15` using a second name with the `rayon` feature

Reply:
Great!